src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.ConvertToString.liquid

```diff
@@ -20,9 +20,9 @@
     else if (value is bool) {
         return System.Convert.ToString(value, cultureInfo).ToLowerInvariant();
     }
-    else if (value is byte[])
```

Comment:
Please keep this as is because there are some users which cannot use these C# 7 features...

Reply:
Done. I note this method has multiple other possible optimizations, but I'll reserve those for another time.